### ISSUE TITLE
Oddly Resizing "References > Table of Contents" Popup Menu

### ISSUE THREAD

**Issue by Tex2002ans** [CollaboraOnline/online#6891]
### Describe the Bug

If you go to:

- "References" tab
- Press "Table of Contents and Index" button.

The popup menu changes size/height depending on which tab you are in.

### Steps to Reproduce

1. Go to the "References" tab.

2. Press "Paragraph" button.

3. Then, click each of the 5 tabs along the top:

- Type
- Entries
- Styles
- Columns
- Background

**BUG: See the entire menu resizing its height ~2 or 3 times as you click along the tabs.**

4. Then, click along those same 5 tabs again.

See the menu's shape/height now stays the same.

### Expected Behavior

Do not erratically resize as you are changing tabs.

### Actual Behavior

Menu changes shape/size... but then after going through each tab, is stable.

### Desktop

> COOLWSD version: 23.05.1.2 (git hash: 6f34565a)
> LOKit version: Collabora Office 23.05.1.2 (git hash: 9a20ef2)
> Served by: Linux Mint 21.1
> Server ID: ab0fb16d

**Comment by timur-g** [comment]
True, and also Background is at first in 2nd row, where it should not be, but after clicks goes to 1st row.
![image](https://github.com/CollaboraOnline/online/assets/7240617/072879b2-929b-4c79-88c5-ed4cfaa6402d)

**Comment by jazevedo-coll** [comment]
This is still happening on 25.04

**Comment by smehrbrodt** [comment]
Fixed with https://gerrit.collaboraoffice.com/plugins/gitiles/online/+/59107fc1007efe66e2d7f00310b5bc7bc3984d79%5E%21/
